src/QsFmt/App/Program.fs

```diff
@@ -46,3 +47,6 @@ let main args =
     | :? UnauthorizedAccessException as ex ->
         eprintfn "%s" ex.Message
         4
+    | Failure (msg) ->
+        eprintfn "%s" msg
```

Comment:
This hides the stack trace which may be useful for debugging (and to include in a bug report). I would be fine with leaving this as an unhandled exception, since the default behavior is to print the message, stack trace, and (on Windows) show a message box saying the program stopped working.